test_ghstack.py

```diff
@@ -2005,6 +2007,62 @@ def test_default_branch_change(self) -> None:
 rUP1 Commit 1
 rINI0 Initial commit''')
 
+    # ------------------------------------------------------------------------- #
+
+    def test_labels(self) -> None:
+        def get_labels(n: int) -> List[str]:
```

Comment:
Wow this helper is so convoluted hahahaha

Reply:
it was much simpler when I just added special handling in the REST interface of `github_fake.py`, but since I changed it to use the GraphQL interface, I had to make it use pagination; although I guess since I'm just using it with `assertEquals`, I could just have it pass in the one plus the number of tags it expects